internal/app/ui/cpvareditor/vareditor.go

```diff
@@ -210,6 +220,29 @@ func (v *VarEditor) isCurrentVarInitial(varName string) bool {
 	return v.currentVars().ValueV(varName, dmvars.NullValue) == v.initialVarValue(varName)
 }
 
+func (v *VarEditor) checkForVarIssue(varName, varValue string) bool {
+	is_list, is_string := strings.HasPrefix(varValue, "list("), strings.HasPrefix(varValue, `"`)
+	variable_bad := false
+	enclosing_c := "This shouldnt be empty"
+	if is_string {
+		//we start and end with a quotation mark so we probably shouldnt break
+		enclosing_c = `"`
+		variable_bad = !strings.HasSuffix(varValue, `"`)
+	} else if is_list {
+		//if list is enclosed
+		enclosing_c = `)`
+		variable_bad = !strings.HasSuffix(varValue, enclosing_c)
+	}
+
+	if variable_bad {
+		dialog.Open(dialog.TypeInformation{
+			Title:       "Variable Error!",
+			Information: "Variable " + varName + ", of value\n" + varValue + "\nis not enclosed with a " + enclosing_c + " !",
+		})
+	}
+	return variable_bad
```

Comment:
In Go we use `camelCase` style to name vars. Please address it.

Reply:
addressed